Data-base/HOME WORK WEEK 1.sql

```diff
@@ -0,0 +1,51 @@
+-- Active: 1724582665781@@127.0.0.1@3308@hyf_lesson1
+Select COUNT(*) as total_tasks from task ;
+/*Find out how many tasks are in the task table*/
+
+SELECT COUNT(*) AS invalid_due_date_count FROM task WHERE due_date IS NULL;OR due_date = '';
+/*Find out how many tasks in the task table do not have a valid due date*/
+
+SELECT *
+FROM task
+WHERE status_id = 'Done';
+/*Find all the tasks that are marked as done*/
+
+
+SELECT *
+FROM task
+WHERE status_id <> 'Done';
```

Comment:
wrong from the same reason as the previous query

Reply:
the same i did it changes